qiskit/wrapper/_wrapper.py

```diff
@@ -364,3 +364,26 @@ def load_qasm_file(qasm_file, name=None,
         QISKitError: if the file cannot be read.
     """
     return circuit_from_qasm_file(qasm_file, name, basis_gates)
+
+
+def qobj_to_quantum_circuit(qobj):
+    """Return a list of QuantumCircuit object(s) from a qobj
+
+    Args:
+        qobj (Qobj): The Qobj object to convert to QuantumCircuits
+    Returns:
+        list: A list of QuantumCircuit objects from the qobj
+
+    """
+    if hasattr(qobj.header, 'compiled_circuit_qasm'):
```

Comment:
I think this `if` never evaluates to True. The `compiled_circuit_qasm` is part of the experiment-level headers, not qobj-level header. So actually this if could be removed and it would still work.

Reply:
Ok, I'll remove it. I wasn't sure and I left this around after my first local draft which had the function operate recursively (which I changed because of python)